pandas/io/parsers.py

```diff
@@ -364,6 +367,7 @@ def parser_f(filepath_or_buffer,
                  converters=None,
                  dtype=None,
                  usecols=None,
+                 skipcols=None,
```

Comment:
@jreback It looks like after I add this line, that error will occur. It must have something to do with the c code, right?